pkg/devcontainer/run.go

```diff
@@ -316,6 +315,17 @@ func runInitializeCommand(
 	return nil
 }
 
+func (r *runner) recreateCustomDriver(ctx context.Context, options UpOptions) (*config.Result, error) {
+	err := r.Driver.StopDevContainer(ctx, r.ID)
+	if err != nil {
+		return nil, errors.Wrap(err, "rebuilding the workspace is currently not supported for this driver")
```

Comment:
not sure the error message makes sense now 🤔

Reply:
Welp in case we will have other custom drivers where recreate doesn't work :shrug:

Reply:
but it could also fail for unrelated reasons, because all custom drivers can be stopped

Reply:
sounds good

Reply:
done :+1: